test/helpers/IntegrationHelper.js

```diff
@@ -826,35 +867,46 @@ global.integration = function (definitions) {
 
                     // Advance the phases to the specified
                     this.advancePhases(options.phase);
-                }
 
-                // Player stats
-                this.player1.damageToBase = options.player1.damageToBase ?? 0;
-                this.player2.damageToBase = options.player2.damageToBase ?? 0;
+                    // Player stats
+                    this.player1.damageToBase = options.player1.damageToBase ?? 0;
```

Comment:
Could we handle this inside the base setup method instead of needing to manage it explicitly here? I feel like that'd be cleaner and easier to maintain

Reply:
yessir!

Reply:
Now that I have tried this quite a few times, it seems that this.player1.damageToBase needs to be set here; otherwise, some tests fail to end the game. I assume it has something to do with which player1 or player2 variable we are setting here. If you want you can take a look at it.